### DISCUSSION TITLE
Touch Sensitivity for Virtual Piano

### DISCUSSION THREAD

**Discussion by xXGoziXx** [jazz-soft/JZZ-input-Kbd#2] [Ideas]
Hey there,
I was wondering if you ever planned on adding touch sensitivity to the virtual keyboard.
I was thinking of doing it but then figured I'd ask here first.
I imagine this can be done by reading the position of the mouse relative to the key and using percentages of the height of the key change the volume in the midi output.

P.S. I'd be happy to do it and contribute to the project as I'd really love to use it in another project I'm working on at the moment.

**Comment by jazz-soft** [comment]
I guess you are talking about something different from the multitouch (that is already supported)?

**Reply by xXGoziXx** [reply]
Yes, I'm talking about volume sensitivity.
So for example tapping the upper part of the key will be a quite sound but the lower part a louder sound.

**Reply by jazz-soft** [reply]
Good idea, but at the moment, I'm not really sure if this feature would worth the amount of work...
You are welcome to fork the repo, and let's see how it will go from there...
